src/modules/eodiro-axios.ts

```diff
@@ -50,6 +56,10 @@ export default async function eodiroAxios<T = any>(
         config.headers.accessToken = accessToken
       }
 
+      if (accessIfExist && accessToken != null) {
+        config.headers.accessToken = accessToken
+      }
```

Comment:
Never use shallow compare. Use `!==` instead.
Also, if there is no `accessToken`, its value may be `undefined` rather than `null`.

Reply:
done!